Library/Formula/haproxy.rb

```diff
@@ -30,4 +29,8 @@ def install
     man1.install "doc/haproxy.1"
     bin.install "haproxy"
   end
+
+  test do
+    system bin/"haproxy", "-v"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!